com.unity.render-pipelines.high-definition/Runtime/Lighting/AtmosphericScattering/AtmosphericScattering.hlsl

```diff
@@ -277,6 +277,7 @@ void EvaluateAtmosphericScattering(PositionInputs posInput, float3 V, out float3
 
         if (_EnableVolumetricFog != 0)
         {
+            bool doBiquadraticReconstruction = _VolumetricFilteringEnabled < 0.5f; // Only if filtering is disabled.
```

Comment:
This is weird, just compare it with 0. More efficient as well.

Reply:
You could also pass an int.

Reply:
This is actually a leftover while I was experimenting, will change!